packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.html

```diff
@@ -2,14 +2,12 @@
   <host>
     <article class="card">
       <header class="slds-grid slds-p-horizontal_x-small slds-has-flexi-truncate">
-        <div class="slds-media__body">
-          <div class="slds-grid card__header">
+        <div class="slds-media__body" onclick={toggleVisibility}  onkeydown={onKeyDown} role="button" tabindex="0">
```

Comment:
```suggestion
        <div class="slds-media__body" onclick={handleHeaderClick} onkeydown={handleHeaderKeyDown} role="button" tabindex="0">
```

Reply:
if possible, I recommend replacing this with a `button` element. This way, you wouldn't need `onkeydown`.